package.json

```diff
@@ -38,6 +38,7 @@
   "devDependencies": {
     "@types/jest": "^27.0.3",
     "@types/jest-dev-server": "^5.0.0",
+    "@types/node": "^16.11.13",
```

Comment:
Why is this here now?

Reply:
Because more than one package uses those typings, so it makes more sense that we put it in the root package.json

Reply:
I'd put them in the engine package instead. Rigth now the server package as mostly all dependencies, but I'll make a cleanup and put the correct deps in the engine and channels package